praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def reply(self, body, author_hidden=False, internal=False):
+        """Reply to the conversation.
+
+        :param body: The markdown formatted content for a message.
+        :param author_hidden: When True, author is hidden from non-moderators.
```

Comment:
Can you please list the default like `(default: False).` at the end of the description?